package.json

```diff
@@ -24,9 +24,10 @@
     "url": "https://github.com/bigcommerce/bigcommerce-api-node.git"
   },
   "dependencies": {
-    "axios": "^0.26.1",
+    "axios": "^0.27.2",
     "form-data": "^4.0.0",
-    "jsonwebtoken": "^8.5.1"
+    "json5": "^1.0.2",
```

Comment:
I don't believe this is a direct dependency so we don't want to add it to the `package.json`. There are already packages in `package.lock` requiring version `^2.1` so I don't think we need to do anything here.

Reply:
Removed it!